### DISCUSSION TITLE
MacBook running docker reports an error

### DISCUSSION THREAD

**Discussion by allenzoe** [docker-library/wordpress#905] [General]
system: MacOS
Use Docker Desktop to pull the latest WordPress image, and then run it directly. Some errors are reported, as follows:
2024-06-24 09:20:09 Error: This does not seem to be a WordPress installation.
2024-06-24 09:20:09 The used path is: /var/www/html/
2024-06-24 09:20:09 Pass --path=`path/to/wordpress` or run `wp core download`.

**Comment by yosifkit** [comment]
That sounds like you pulled `wordpress:cli-2` and not `wordpress:6.5` (with more or less version or PHP specificity). The `cli` image only has the `wp` cli and just runs the `wp shell` command. You have to point its volume at an already installed set of WordPress files, use `wp` to install WordPress to the directory, or use the non-cli image to initialize the volume directory.
